queue/src/fiskaltrust.Middleware.Localization.QueueIT/v2/Lifecycle/OutOfOperationReceipt0x4002.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using fiskaltrust.ifPOS.v1;
+using fiskaltrust.ifPOS.v1.it;
+using fiskaltrust.Middleware.Contracts.Extensions;
+using fiskaltrust.Middleware.Localization.QueueIT.Constants;
+using fiskaltrust.storage.V0;
+
+namespace fiskaltrust.Middleware.Localization.QueueIT.v2.Lifecycle
+{
+    public class OutOfOperationReceipt0x4002 : IReceiptTypeProcessor
+    {
+        private readonly IITSSCD _itSSCD;
+        private readonly IConfigurationRepository _configurationRepository;
+
+        public ITReceiptCases ReceiptCase => ITReceiptCases.OutOfOperationReceipt0x4002;
+
+        public OutOfOperationReceipt0x4002(IITSSCD itSSCD, IConfigurationRepository configurationRepository)
+        {
+            _itSSCD = itSSCD;
+            _configurationRepository = configurationRepository;
+        }
+
+        public async Task<(ReceiptResponse receiptResponse, List<ftActionJournal> actionJournals)> ExecuteAsync(ftQueue queue, ftQueueIT queueIt, ReceiptRequest request, ReceiptResponse receiptResponse, ftQueueItem queueItem)
+        {
+            if (queue.IsDeactivated())
```

Comment:
Also already checked by the SignProcessor.

Reply:
same